pages/login.tmpl

```diff
@@ -42,14 +46,17 @@ hr.short {
 		{{range .Flashes}}<li class="urgent">{{.}}</li>{{end}}
 	</ul>{{end}}
 
-	{{ if or .OauthSlack .OauthWriteAs }}
+	{{ if or .OauthSlack .OauthWriteAs .OauthGitlab }}
```

Comment:
I'm not sure if the `or` template function accepts more than one parameter. It may have to be rewritten as:

```
{{ if or (or OauthSlack .OauthWriteAs) .OauthGitlab }}
```

Reply:
It seems to work as expected at the moment - if any one of the three are set, it shows the option